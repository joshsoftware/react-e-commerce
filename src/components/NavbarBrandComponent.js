import React from 'react';
import { NavbarBrand } from 'reactstrap';
import PropTypes from 'prop-types';
import ImageComponent from './ImageComponent';

const NavbarBrandComponent = ({ logo }) => {
  return (
    <>
      <NavbarBrand>
        <ImageComponent style={{ width: '70px' }} src={logo} className="rounded-circle" alt="img" />
      </NavbarBrand>
    </>
  );
};

export default NavbarBrandComponent;

NavbarBrandComponent.propTypes = {
  url: PropTypes.string.isRequired,
  logo: PropTypes.string.isRequired
};
