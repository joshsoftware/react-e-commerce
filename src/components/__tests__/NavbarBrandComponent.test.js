import NavbarBrandComponent from '../NavbarBrandComponent';
import { render } from '@testing-library/react';
import React from 'react';

describe('NavbarBrand Component', () => {
  it('Must return Navigation bar brand', () => {
    const url = '/';
    const logo =
      'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png';
    const { asFragment } = render(<NavbarBrandComponent url={url} logo={logo} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
