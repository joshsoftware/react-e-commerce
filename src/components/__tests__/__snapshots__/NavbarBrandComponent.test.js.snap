// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarBrand Component Must return Navigation bar brand 1`] = `
<DocumentFragment>
  <a
    class="navbar-brand"
    href="/"
  >
    <img
      alt="img"
      class="rounded-circle"
      src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png"
      style="width: 70px;"
    />
  </a>
</DocumentFragment>
`;
